pandas/core/apply.py

```diff
@@ -408,6 +414,77 @@ def agg_list_like(self) -> DataFrame | Series:
             )
             return concatenated.reindex(full_ordered_index, copy=False)
 
+    def future_list_single_arg(
```

Comment:
same request from the other PR re naming; "future" won't be very helpful for a reader a year from now

Reply:
The intention is to have "future_" methods alongside the current methods; all with the same prefix so they are easy to identify. Any such method is behind the option "future_udf_behavior" meaning they will only be called when set to True. Assuming we do end up going forward with this new (experimental) behavior, once it is in a good place we deprecate the option and then remove the option.

A year from now, we will still have the option "future_udf_behavior", and in my opinion, the "future_" prefix is meaningful and helpful. When the option is removed, the "old" methods are removed and the "future_" methods are renamed by removing the prefix (none are public).

Reply:
you've clearly given this more thought than i have so im going to stop complaining about this, will instead grumble to myself

Reply:
The name is still up for improvements and suggestions are most welcome, but wanted to explain why I felt "future" was appropriate.